src/ecs/systems/client/HandleIncomingMessages.hpp

```diff
@@ -76,26 +76,24 @@ namespace ecs::systems
                     world.registry.addComponent<ecs::component::Shootable>(
                         newEntity, ecs::component::Shootable(sf::Keyboard::Space));
                     world.registry.addComponent<ecs::component::Controllable>(
-                        newEntity, {sf::Keyboard::Z, sf::Keyboard::Q, sf::Keyboard::S, sf::Keyboard::D});
+                        newEntity, {sf::Keyboard::Z, sf::Keyboard::Q, sf::Keyboard::S, sf::Keyboard::D, sf::Keyboard::H});
+                    world.registry.addComponent<ecs::component::Hitbox>(
+                        newEntity, {ecs::component::Hitbox()});
                 }
                 world.registry.addComponent<component::Drawable>(newEntity, {"players", {1, 1, 32, 16}});
                 break;
             case component::EntityType::Types::EnemyBase:
                 world.registry.addComponent<component::Drawable>(newEntity, {"players", {1, 18, 32, 16}});
+                world.registry.addComponent<ecs::component::Hitbox>(
+                    newEntity, {ecs::component::Hitbox()});
```

Comment:
you could just have passed {} instead of {ecs::component::Hitbox()}

Reply:
true

Reply:
Nop